app/views/govuk_publishing_components/components/_intervention.html.erb

```diff
@@ -30,6 +30,10 @@
   intervention_helper = GovukPublishingComponents::Presenters::InterventionHelper.new(options)
   dismiss_href = intervention_helper.dismiss_link
 
+  ga4_tracking ||= false
+  suggestion_data_attributes[:module] = suggestion_data_attributes[:module] ? suggestion_data_attributes[:module] + " ga4-link-tracker" : "ga4-link-tracker" if ga4_tracking
+  suggestion_data_attributes[:ga4_link] = { event_name: "navigation", type: "intervention", section: suggestion_text, index: 1, index_total: 1 }.to_json if ga4_tracking
```

Comment:
@andysellick We have `data-ga4-set-indexes` which does set indexes using JavaScript. However I believe it only works in conjuction with `data-ga4-track-links-only`. However there is no point using `data-ga4-track-links-only` here, as we are setting `ga4-link-tracker` directly on the link, so I may as well set the index on it since it's the only link.